cmd/multiFaReduce/multiFaReduce.go

```diff
@@ -0,0 +1,79 @@
+// Command Group: "FASTA and Multi-FASTA Tools"
+
+package main
+
+import (
+	"flag"
+	"fmt"
+	"github.com/vertgenlab/gonomics/dna"
+	"github.com/vertgenlab/gonomics/fasta"
+	"log"
+)
+
+func makeEmptyFastas(records []fasta.Fasta) []fasta.Fasta {
```

Comment:
I think there is a function in fasta called SegregatingSites that performs a similar or identical task. We might want to see if we can consolidate to reduce redundancy

Reply:
good point.  we don't want to maintain two versions of the same thing.  i will look into this.

Reply:
I have now fixed this and call the existing function (I have eliminated the duplicate that I wrote).